Mathlib/Order/Interval/Finset/Nat.lean

```diff
@@ -139,22 +146,14 @@ lemma mem_Ioc_succ : a ∈ Ioc b (b + 1) ↔ a = b + 1 := by simp
 lemma mem_Ioc_succ' (a : Ioc b (b + 1)) : a = ⟨b + 1, mem_Ioc.2 (by omega)⟩ :=
   Subtype.val_inj.1 (mem_Ioc_succ.1 a.2)
 
+set_option linter.deprecated false in
 theorem Ico_succ_right_eq_insert_Ico (h : a ≤ b) : Ico a (b + 1) = insert b (Ico a b) := by
   rw [Ico_succ_right, ← Ico_insert_right h]
 
+set_option linter.deprecated false in
 theorem Ico_insert_succ_left (h : a < b) : insert a (Ico a.succ b) = Ico a b := by
   rw [Ico_succ_left, ← Ioo_insert_left h]
```

Comment:
I assume you're planning to deal with these in a future PR? If so, let's have a TODO here too, and if not let's have a comment explaining the inconsistency.

Reply:
or the third option :)